src/ros_numpy/image.py

```diff
@@ -80,6 +80,9 @@ def image_to_numpy(msg):
 		data = data[...,0]
 	return data
 
+@converts_to_numpy(CompressedImage)
+def compressed_image_to_numpy(msg):
+	return np.fromstring(msg.data, np.uint8)
```

Comment:
About frombuffer, done, commited.

About the CompressedImage, the reality of compressed images is not very nice. That transformation works for all jpg. For png one can do exactly the same but must skip the first 12 bits of a header. For CompressedImages that contain a depth image (there is no way to know) there is one step more even... So that would be a bit of a mess to figure out all the cases.
I think this implementation helps, better than none.

Reply:
>  For png one can do exactly the same but must skip the first 12 bits of a header

Huh? What do you mean by this?

> So that would be a bit of a mess to figure out all the cases.

Then throw an error in the cases that aren't supported.

> I think this implementation helps, better than none.

I disagree - the point of this library was to do the "one obvious thing". The obviously thing for a comrpessed image is to return a 3D RGB array. Returning a 1d sequence of bytes is not what I would expect at all.